assets/src/hooks/v2/use_api_response.tsx

```diff
@@ -163,8 +164,8 @@ const useBaseApiResponse = ({
   let refreshRateOffsetMs = parseInt(refreshRateOffset, 10) * 1000;
   let apiPath = `/v2/api/screen/${id}${routePart}?last_refresh=${lastRefresh}${isRealScreenParam}${screenSideParam}${requestorParam}`;
 
-  if (isDup()) {
-    apiPath = `https://screens.mbta.com${apiPath}&rotation_index=${ROTATION_INDEX}`;
+  if (isOFM()) {
+    apiPath = `http://localhost:4000${apiPath}&rotation_index=${ROTATION_INDEX}&pane=${TRIPTYCH_PANE}`;
```

Comment:
Whoops, accidentally committed this change from an earlier attempt at packaging. I'll reverse the localhost url back to screens.mbta.com.

Also, I think I will need to conditionally add the `rotation_index` and `pane` params instead of always adding both for either OFM app—it would make the logs kind of confusing if both were included for any DUP/triptych request.